README.md

```
@@ -1,25 +1,163 @@
-# Twiggle - Urban Garden Planner (Backend)
+# 🌱 Twiggle - Your Gardening Companion
+
+<div align="center">
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: div

(MD033, no-inline-html)